### PR TITLE
Fixes broken link CSS on the homepage

### PR DESCRIPTION
**Pull request by sarahjschultz**
### SUMMARY:
Fixes https://github.com/sarahjschultz/kittengif/issues/4. 

#### BAD:
screen shot 2016-11-11 at 2 54 17 pm

#### GOOD:
screen shot 2016-11-13 at 8 11 25 pm

### CODE REVIEW

File: stylesheets/main.css [hunk 1]
```diff
@@ -12,6 +12,16 @@ body {
   color: white;
 }
 
+.about-link a {
+  text-decoration: none;
```
Comment (carlwiedemann) [inline stylesheets/main.css]: This line isn't necessary since it will cascade down from `a`

File: stylesheets/main.css [hunk 2]
```diff
@@ -12,6 +12,16 @@ body {
...
+  color: white;
+}
+
+.about-link a:hover {
+  text-decoration: none;
```
Comment (carlwiedemann) [inline stylesheets/main.css]: ditto re: not necessary

**Comment by carlwiedemann** [comment]
No blockers 👍

File: stylesheets/main.css (hunk 2 above)
Comment (sarahjschultz) [reply stylesheets/main.css]: Thanks!

**Comment by carlwiedemann** [comment]
Can you also update the timestamp on the css url in home.html to break caches?

**Comment by carlwiedemann** [comment]
You can use 1479093388

**Comment by carlwiedemann** [comment]
ps, add to your .zshconfig

``` sh
alias ts='date -j -f "%a %b %d %T %Z %Y" "`date`" "+%s"'
```

Then when you type 'ts' on the commandline you will get a timestamp

**Comment by carlwiedemann** [comment]
^^ You'll also need to `source ~/.zshconfig` after you make this change 😸

**Comment by carlwiedemann** [comment]
🇨🇦

**Comment by sarahjschultz** [comment]
@c4rl 🇮🇴

**Pull request merged**